app/addons/auth/stores/ChangePassword.js

```diff
@@ -0,0 +1,57 @@
+import FauxtonAPI from "../../../core/api";
+import {
+  AUTH_CLEAR_CHANGE_PWD_FIELDS,
+  AUTH_UPDATE_CHANGE_PWD_FIELD,
+  AUTH_UPDATE_CHANGE_PWD_CONFIRM_FIELD
+} from "../actiontypes";
+
+const ChangePasswordStore = FauxtonAPI.Store.extend({
```

Comment:
If you feeling brave, you are welcome to change this all to use redux. See the permission section as an example